Cause warning "Trailing spaces not allowed" when compiling in MacOS  

<!--

We use GitHub only for bug reports 🐛

Anything else should be posted to https://discuss.redash.io 👫

🚨For support, help & questions use https://discuss.redash.io/c/support
💡For feature requests & ideas use https://discuss.redash.io/c/feature-requests

**Found a security vulnerability?** Please email security@redash.io to report any security vulnerabilities. We will acknowledge receipt of your vulnerability and strive to send you regular updates about our progress. If you're curious about the status of your disclosure please feel free to email us again. If you want to encrypt your disclosure email, you can use this PGP key.

-->

### Issue Summary

When I compiling in MacOS, I got a "Trailing spaces not allowed" warning, Details are as follows:

WARNING in ./client/app/components/QueryEditor.jsx
Module Error (from ./node_modules/eslint-loader/index.js):

/Users/simon/repository/redash/client/app/components/QueryEditor.jsx
  128:1  error  Trailing spaces not allowed  no-trailing-spaces

✖ 1 problem (1 error, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.

### Steps to Reproduce

during "npm run start" command running.

### Technical details:

* Redash Version: 6.0
* Browser/OS: MacOS 10.14.2
* How did you install Redash: Docker


After I deleted line 128 and recompiled again, everything OK, Compiled successfully.

--- a/client/app/components/QueryEditor.jsx
+++ b/client/app/components/QueryEditor.jsx
@@ -125,7 +125,7 @@ class QueryEditor extends React.Component {
     editor.commands.bindKey('Cmd+L', null);
     editor.commands.bindKey('Ctrl+P', null);
     editor.commands.bindKey('Ctrl+L', null);
-            
+
     // Ignore Ctrl+P to open new parameter dialog
     editor.commands.bindKey({ win: 'Ctrl+P', mac: null }, null);
     // Lineup only mac

